node_modules/oae-mixpanel/lib/api.content.js

```diff
@@ -0,0 +1,124 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var Validator = require('oae-util/lib/validator').Validator;
+
+var MixPanelUtil = require('./util');
+
+/**
+ * Get the total size of uploaded files for the current month for all tenants
+ * or the current tenant, depending on the used router.
```

Comment:
I see what's going on. Let's alter the API layout a bit if possible:

1. Don't have variations for `ForTenant` and having a tenant
2. Have all stats methods take in a `tenantAlias`
3. If `alias` is not specified, default it to the current tenant
4. All other logic is the same thereon (the current user in context must be able to administer the specified tenant, etc...)